hack/devtools/local_dev_env.sh

```diff
@@ -39,13 +39,42 @@ ask_to_create_default_env_config() {
     fi
 }
 
+# We use a service principal and certificate as the mock MSI object
+create_mock_msi() {
+    appName="mock-msi-$(openssl rand -base64 9 | tr -dc 'a-zA-Z0-9' | head -c 6)"
+    az ad sp create-for-rbac --name $appName --create-cert --output json
+}
+
+get_mock_msi_clientID() {
+    echo "$1" | jq -r .appId
+}
+
+get_mock_msi_tenantID() {
+    echo "$1" | jq -r .tenant
+}
+
+get_mock_msi_cert() {
+    certFilePath=$(echo "$1" | jq -r '.fileWithCertAndPrivateKey')
+    base64EncodedCert=$(base64 -w 0 $certFilePath)
+    rm $certFilePath
+    echo $base64EncodedCert
+}
+
 create_env_file() {
```

Comment:
Since we are creating Mock MSI under default env file, should we add a prompt to see if the user is planning to create a MIWI ARO RP or Regular RP for Dev?

Reply:
I'll add this when we merge to master